node/src/main/resources/propose.conf

```diff
@@ -0,0 +1,33 @@
+leader = {
+    host = "localhost",
+    grpc-port = 40402,
+    http-port=40403,
+    propose-info= {
+        deploy-contract: "Nil",
+        phlo-limit: 100,
+        phlo-price: 1,
+        deploy-private-key: "34d969f43affa8e5c47900e6db475cb8ddd8520170ee73b2207c54014006ff2b"
+    }
+}
+followers = [
+    {host = "localhost",
+     grpc-port = 40302 ,
+     http-port=40403,
+     propose-info= {
+         deploy-contract: "Nil",
+         phlo-limit: 100,
+         phlo-price: 1,
+         deploy-private-key: "5438ac0bde91bb813c7b4b17f215cd6ad52b19e7bcbf1f7907adb1d69c8aa7b1"
+     }},
+     {host = "localhost",
+     grpc-port = 40202 ,
+     http-port=40403,
+     propose-info= {
+         deploy-contract: "Nil",
```

Comment:
I think it would be very useful to have here folder configurable and read random `.rho` file, to test merging.